src/ex4/dist/app.js

```diff
@@ -0,0 +1,102 @@
+const addButton = document.getElementById("list-item-submit")
+const textInput = document.getElementById("list-item-input")
+const allLists = document.getElementById("list")
+const counterLists = document.getElementById("counter")
+const errorMessage = document.getElementById("error")
+const regex = /^[0-9]*$/
+
+const itemC = new ItemClient()
+
+class Main {
+  constructor() {
+    this.name = "Main"
+  }
+  init() {
+    addButton.addEventListener("click", function () {
+      validation()
+    })
+  }
+}
+function addNewList(name) {
+  const li = document.createElement("li")
```

Comment:
what happened to the DOM manager class from previous exercises? separation of concerns is important